imessage2020wrapped/sql_queries.py

```diff
@@ -218,22 +257,28 @@
     message_date DESC;
 """
 
-AVG_MESSAGE_LENGTH = """
+def avg_message_length(year):
+    return """
 SELECT
     AVG(LENGTH(message.text)),
     datetime (message.date / 1000000000 + strftime ("%s", "2001-01-01"), "unixepoch", "localtime") AS message_date,
     message.is_from_me,
-    chat.chat_identifier as number
+    chat.chat_identifier as number,
+    date (date / 1000000000 + strftime ("%s", "2001-01-01"), "unixepoch", "localtime") as day
 FROM
     message
 JOIN chat_message_join cmj on cmj.message_id = message.ROWID
 JOIN chat on chat.ROWID = cmj.chat_id
 WHERE
-    chat_identifier REGEXP '\+1\d{10}'
+    chat_identifier REGEXP '\+1\d{{10}}'
```

Comment:
Did you mean to change regex?

Reply:
When you use `.format` you have to double-quote braces so the formatter doesn't confuse them with variable interpolations like I do below in `{year}` and `{next_year}`: https://stackoverflow.com/a/5466478.

Reply:
Ah yep. Makes sense. Merging now.